apps/reference/docs/guides/realtime/limits.mdx

```diff
@@ -0,0 +1,66 @@
+---
+id: limits
+title: Realtime Limits
+description: Realtime Rate Limiting
+---
+
+Since deploying a global multi-tenant cluster for Supabase Realtime, we've implimented some rate limits to protect our infrastructure.
+
+Rate limits are mostly configurable per project and our cluster can support millions of concurrent connections if needed. If you're running into any rate limiting and it's causing issues, please [contact support](https://app.supabase.com/support/new).
+
+## Limits by Plan
+
+Limits are set on the Free and Pro plans accordingly. See our [pricing page](https://supabase.com/pricing) for more information.
```

Comment:
Perhaps it's better to write:

```suggestion
There are default limits on the Free and Pro plans. You can refer to the [pricing page](https://supabase.com/pricing) for more information on those limits.
```

Reply:
There are default limits on the Free and Pro plans. Refer to the [pricing page](https://supabase.com/pricing) for more information.

Reply:
"default" implies we'll increase your limits on one of those plans. 

If you're on the free plan and you need more Realtime throughput you need to upgrade.